lti_consumer/lti_1p1/exceptions.py

```diff
@@ -0,0 +1,9 @@
+"""
+Exceptions for the LTI Consumer.
+"""
+
+
+class LtiError(Exception):
+    """
+    General error class for LTI Consumer usage.
+    """
```

Comment:
I think this should be a common error, and this particular error can be Lti1p1Error?

Reply:
I'm not quite sure I follow what you mean by `a common error`? Should I be moving `LtiError` into a separate file, or just removing it altogether and replacing it with `Lti1p1Error`?

Reply:
@pcockwell I think it makes more sense to rename this one to `Lti1p1Error` for LTI 1.1 specific cases, and use a generic LTI error when throwing from `lti_consumer.py` (the block).

Reply:
@giovannicimolin I've pushed some changes. Please let me know if this was your intended result.